### PR TITLE
changelog: Add links to PRs

### PR DESCRIPTION
**Pull request by Hoverbear**
Add links to the changelog so it's easier to see what happened.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # 0.3.0 - 2018-06-07
 
-- Fix deadlock during prevote migration process
-- Keep compatible with protobuf 2.0
-- Make raft `Send`
-- Add more docs and example
-- Support configuring election timeout range
+- Now supports configuring the election timeout range (https://github.com/pingcap/raft-rs/pull/63).
+- Fixed a compiler error (https://github.com/pingcap/raft-rs/pull/64)
```
Comment (BusyJay) [inline CHANGELOG.md]: I think both this and "Workaround a problem" are parts of "keep compatible with rust-protobuf 2.0", which seems more clear.

Comment (Hoverbear) [reply CHANGELOG.md]: :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # 0.3.0 - 2018-06-07
 
-- Fix deadlock during prevote migration process
-- Keep compatible with protobuf 2.0
-- Make raft `Send`
-- Add more docs and example
-- Support configuring election timeout range
+- Now supports configuring the election timeout range (https://github.com/pingcap/raft-rs/pull/63).
```
Comment (siddontang) [inline CHANGELOG.md]: why here use `supports` but later use `made`, `added`? different tense.

Comment (Hoverbear) [reply CHANGELOG.md]: Good point!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+- Fixed a deadlock in the prevote migration process (https://github.com/pingcap/raft-rs/pull/42)
 
 # 0.2.0 - 2018-05-14
 
-- Deprecated sync-log and add context
-- Fix learner isolation bug
-- Port several tests
-- Add examples
-- Use fxhash
+- Deprecated sync-log and add context (https://github.com/pingcap/raft-rs/pull/59)
```
[the remark this answers is not in the document]
Comment (Hoverbear) [reply CHANGELOG.md]: Nice catch!

**Pull request merged**